tensorflow/python/ops/data_flow_ops.py

```diff
@@ -2099,6 +2100,10 @@ def __init__(self,
       file_pattern: File path to the dataset, possibly containing wildcards.
         All matching files will be iterated over each epoch.
       batch_size: How many records to return at a time.
+      minibatches: None by default, otherwise specifies
```

Comment:
I personally find it more natural to think of minibatches as a multiplier of batch_size, rather than a divisor.  batch_size specifies the size of each mini_batch and minibatches tell you how many you get. 

So if you want 4 batches of 32, (batch_size = 32 and minibatches = 4) rather than (batch_size = 128 and minibatches=4).

Thoughts?